### PR TITLE
Changed output filename to remove `:` character

### LINKED ISSUE #5: File cannot be created on Windows due to ":" in filename
## Description

When running the tool, the file cannot be created as `new Date()).toISOString()` results in a filename containing illegal characters (`:`).

## Steps to Reproduce

Execute file `node nr-find-log4j.js`

## Expected Behavior

Script runs and produces CSV/JSON output file.

## Relevant Logs / Console output

```
Checking api key...  OK, found 215 accounts.
Scanning your accounts, this may take some time...
Checking 109 java services... done. Actual service count is 109.
Scanning modules (service 109 of 109)...  done.

OK, scan took 81 seconds. Found 15 services with log4j-core.
node:fs:585
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open 'log4j_scan_2021-12-14T13:30:13.870Z.csv'
    at Object.openSync (node:fs:585:3)
    at Object.writeFileSync (node:fs:2153:35)
    at writeResults (D:\git\nr-find-log4j\nr-find-log4j.js:542:8)
    at findModulesByEntity (D:\git\nr-find-log4j\nr-find-log4j.js:367:3)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async findServices (D:\git\nr-find-log4j\nr-find-log4j.js:281:5)
    at async D:\git\nr-find-log4j\nr-find-log4j.js:199:7 {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: 'log4j_scan_2021-12-14T13:30:13.870Z.csv'
}
```

## Your Environment

Windows 11 
Tested on Node 14, Node 16

## Additional context

n/a

### PR DESCRIPTION
**Pull request by karl-barbour**
Fixes #5

### CODE REVIEW

**Review by fightingmonk — CHANGES_REQUESTED** [review]
Thank you for the quick PR to fix this!

I tested a few environments and found that `replaceAll()` isn't supported on my version of node v12 or v14. I suggest using `.replace(/:/g, '-')` instead.

File: nr-find-log4j.js [hunk 1]
```diff
@@ -429,7 +429,7 @@ function writeResults(state) {
     state.scanDurationSec = Math.ceil((state.scanCompleted - state.scanStarted) / 1000);
 
     process.stdout.write(`\nOK, scan took ${state.scanDurationSec} seconds. Found ${vulnerableApplications.length} services with log4j-core.\n`);
-    const fileTimestamp = (new Date()).toISOString();
+    const fileTimestamp = new Date().toISOString().replaceAll(":","_");
```
Comment (fightingmonk) [inline nr-find-log4j.js]: string.replaceAll() isn't supported in the Node v12 and v14 versions I tested with. we could use the regex global version of replace instead: `.replace(//g, '')`

Comment (karl-barbour) [reply nr-find-log4j.js]: You're absolutely correct, apologies @fightingmonk - I had upgraded to 16 to see if it fixed the initial issue and didn't test backwards compatibility. New change suggested.

**Review by fightingmonk — APPROVED** [review]
perfect, thanks for the fix!

**Pull request merged**